client/js/components/statement/publicStatementLists/DpPublicStatementList.vue

```diff
@@ -13,12 +13,12 @@
       v-if="transformedStatements.length === 0"
       :message="Translator.trans('statement.list.empty')"
       type="info" />
-    <draggable
+    <dp-draggable
       v-else
-      v-model="transformedStatements"
+      v-model:content-data="transformedStatements"
```

Comment:
If I remember correctly. v-model doesn't work with the new implementation of DpDraggable. So it should just be
```
:content-data="transformedStatements"
```
While we don't need to fix draggable right now, we could still change this in case we need to fix it in the future.